src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/vedtaksperiode/produsent/GrunnlagForPerson.kt

```diff
@@ -81,6 +81,30 @@ data class VilkårResultatForVedtaksperiode(
     )
 }
 
+fun List<VilkårResultatForVedtaksperiode>.sammenlignUtenFomOgTom(other: List<VilkårResultatForVedtaksperiode>): Boolean {
+    return this.map { ComparableVilkårResultat(it) }.toSet() == other.map { ComparableVilkårResultat(it) }.toSet()
```

Comment:
Blir dette det samme?
```suggestion
    return this.map { it.copy(fom = null, tom = null) }.toSet() == other.map { it.copy(fom = null, tom = null) }.toSet()
```

Reply:
Så slipper vi å lage en egen klasse

Reply:
Har du lyst til å fikse dette på linje 67 og 90 også? 👼